site/docs/concepts/config-policies.md

```diff
@@ -215,57 +215,45 @@ config[{"capability": "copy", "decision": decision}] {
+Rules are written in the following syntax: `optimize[decision]` where
 
-For example, the following attribute defines the storage cost of the "account-theshire" storage account. 
+- `decision` is a JSON structure with the following fields:
+- `policy` - policy metadata: unique ID, human-readable description and a version
+- list of `goals` including attribute name, optimization directive(`min` or `max`) and optionally a weight.
+
+For example, the following rule attempts to minimize storage cost in copy scenarios.
 
 ```
-{
-    "attribute": "storage-cost",
-    "description": "theshire object store",
-    "value": "90",
-    "type": "numeric",
-    "units": "US Dollar per TB per month",
-    "object": "storageaccount",
-    "instance": "account-theshire"
+# minimize storage cost for copy scenarios
+optimize[decision] {
+    input.request.usage == "copy"
+    policy := {"ID": "save-cost", "description":"Save storage costs", "version": "0.1"}
+    decision := {"policy": policy, "strategy": [{"attribute": "storage-cost", "directive": "min"}]}
 }
 ```
 
-### Add a new attribute definition to the taxonomy
-
-See https://github.com/fybrik/fybrik/blob/master/samples/taxonomy/example/infrastructure/attributepair.yaml for an example how to define an attribute and the corresponding measurement units. 
- 
-### Usage of infrastructure attributes in policies
+### Weights
 
-An infrastructure attribute can be used as the `property` value in configuration policies. For example, the following policy restricts 
-the storage account selection using the `storage-cost` infrastructure attribute:
+If more than one goal is provided, they can have a different weight. By default, all weights are equal to 1. 
+For example, the rule below defines two goals with weights in the 4:1 ratio meaning that the optimizer try to optimize distance and storage costs, but will give a higher priority to distance.
 ```
-# restrict storage costs to a maximum of $95 when copying the data
-config[{"capability": "copy", "decision": decision}] {
-    input.request.usage == "copy"
-    input.request.dataset.geography != input.workload.cluster.metadata.region
-    account_restrict := {"property": "storage-cost", "range": {"max": 95}}
-    policy := {"ID": "copy-restrict-storage", "description":"Use cheaper storage", "version": "0.1"}
-    decision := {"policy": policy, "restrictions": {"storageaccounts": [account_restrict]}}
+# minimize distance, minimize storage cost for read scenarios
+optimize[decision] {
+    input.request.usage == "read"
+    policy := {"ID": "general-strategy", "description":"focus on higher performance while saving storage costs", "version": "0.1"}
+    optimize_distance := {"attribute": "distance", "directive": "min", "weight": "0.8"}
+    optimize_storage := {"attribute": "storage-cost", "directive": "min", "weight": "0.2"}
+    decision := {"policy": policy, "strategy": [optimize_distance,optimize_storage]}
 }
```

Comment:
Should we mention that optimization is turned off by default and link to how to turn it on?

Reply:
Added a link to the optimizer page and instructions how to enable the optimization.